tests/unit/labels/BinaryLabels_unittest.cc

```diff
@@ -66,4 +86,27 @@ TEST(BinaryLabels, serialization)
 		EXPECT_NEAR(labels->get_label(i), new_labels->get_label(i), 1E-15);
 	}
 	unlink(filename);
+
+	SG_UNREF(labels);
+	SG_UNREF(new_labels);
+}
+
+TEST_F(BinaryLabels, set_values_labels_from_constructor)
+{
+	CBinaryLabels* labels = new CBinaryLabels(probabilities);
+
+	SGVector<float64_t> labels_vector = labels->get_labels();
+	SGVector<float64_t> values_vector = labels->get_values();
+
```

Comment:
you could put `ASSERT` here for the pointer not being NULL, and for the size of the vectors to be both equal to N